Kudu.Services/Diagnostics/ProcessController.cs

```diff
@@ -468,7 +468,15 @@ private Process GetProcessById(int id)
         {
             try
             {
-                return id <= 0 ? Process.GetCurrentProcess() : Process.GetProcessById(id);
+                switch (id)
+                {
+                    case 0:
+                        return Process.GetCurrentProcess();
+                    case -1:
+                        return Process.GetProcessesByName("w3wp").FirstOrDefault(s => s.Id != Process.GetCurrentProcess().Id) ?? Process.GetCurrentProcess();
```

Comment:
So it's the Main site, but if Main is not running you return the scm? Seems a bit funky. Needs comment to explain reasoning :)